APMrover2/GCS_Mavlink.cpp

```diff
@@ -1450,44 +1440,6 @@ void Rover::gcs_update(void)
     }
 }
 
-void Rover::gcs_send_text(MAV_SEVERITY severity, const char *str)
-{
-    for (uint8_t i=0; i<num_gcs; i++) {
-        if (gcs[i].initialised) {
-            gcs[i].send_text(severity, str);
-    }
-    }
-#if LOGGING_ENABLED == ENABLED
-    DataFlash.Log_Write_Message(str);
```

Comment:
turns out the original GCS_MAVLINK::gcs_send_statustext() didn't log either! Whoops